scipy/stats/_stats_py.py

```diff
@@ -3120,6 +3120,8 @@ def gzscore(a, *, axis=0, ddof=0, nan_policy='propagate'):
     >>> fig, ax = plt.subplots()
     >>> ax.hist(gzscore(x), 50)
     >>> plt.show()
+    
+    Also for the zscore nomalize value you need to use zmap to get the correct zscore values
```

Comment:
I agree this sentence is not very clear. Perhaps add a tiny reproducible example to show what you mean after writing something like

```suggestion
    To obtain the normalized zscore values, you can use the `zmap` function.
```